include/rocksdb/utilities/backupable_db.h

```diff
@@ -274,16 +274,23 @@ class BackupEngineReadOnly {
     return RestoreDBFromLatestBackup(options, db_dir, wal_dir);
   }
 
+  // If with_checksum is false, this function
   // checks that each file exists and that the size of the file matches our
   // expectations. it does not check file checksum.
   //
+  // If with_checksum is true, this function
+  // inspects the current backup file checksum to see if it matches our
+  // expectation.
+  //
   // If this BackupEngine created the backup, it compares the files' current
-  // sizes against the number of bytes written to them during creation.
+  // sizes (resp., current checksum) against the number of bytes written to
+  // them (resp., the checksum calculated) during creation.
   // Otherwise, it compares the files' current sizes against their sizes when
   // the BackupEngine was opened.
   //
   // Returns Status::OK() if all checks are good
-  virtual Status VerifyBackup(BackupID backup_id) = 0;
```

Comment:
Maybe better to use the full name `verify_with_checksum`? It would be more readable in the code.